<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7809" started-at="2015-07-10T10:21:41Z" finished-at="2015-07-10T10:21:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7809" started-at="2015-07-10T10:21:41Z" finished-at="2015-07-10T10:21:48Z">
      <class name="com.example.tests.ContactsCreationTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="setUp" is-config="true" duration-ms="5805" started-at="2015-07-10T13:21:41Z" finished-at="2015-07-10T13:21:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testEmptyContactCreation()[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="testEmptyContactCreation" duration-ms="1509" started-at="2015-07-10T13:21:46Z" finished-at="2015-07-10T13:21:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyContactCreation -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation(com.example.tests.ContacnData)[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="testNonEmptyContactCreation" duration-ms="1" started-at="2015-07-10T13:21:48Z" data-provider="randomValidContactGenerator" finished-at="2015-07-10T13:21:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContacnData [fname=test903457455, lname=test1535041464, hnumber=null, mail1=null]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>

	at com.example.tests.ContactsCreationTests.testNonEmptyContactCreation(ContactsCreationTests.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation(com.example.tests.ContacnData)[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="testNonEmptyContactCreation" duration-ms="0" started-at="2015-07-10T13:21:48Z" data-provider="randomValidContactGenerator" finished-at="2015-07-10T13:21:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContacnData [fname=, lname=test-868068676, hnumber=null, mail1=null]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>

	at com.example.tests.ContactsCreationTests.testNonEmptyContactCreation(ContactsCreationTests.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation(com.example.tests.ContacnData)[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="testNonEmptyContactCreation" duration-ms="0" started-at="2015-07-10T13:21:48Z" data-provider="randomValidContactGenerator" finished-at="2015-07-10T13:21:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContacnData [fname=, lname=test455007132, hnumber=null, mail1=null]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Bound mismatch: The generic method sort(List<T>) of type Collections is not applicable for the arguments (List<ContacnData>). The inferred type ContacnData is not a valid substitute for the bounded parameter <T extends Comparable<? super T>>

	at com.example.tests.ContactsCreationTests.testNonEmptyContactCreation(ContactsCreationTests.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactsCreationTests@4c75cab9]" name="tearDown" is-config="true" duration-ms="481" started-at="2015-07-10T13:21:48Z" finished-at="2015-07-10T13:21:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.ContactsCreationTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
